CoreVisualSchema/src/au/gov/asd/tac/constellation/graph/schema/visual/concept/VisualConcept.java

```diff
@@ -184,6 +184,11 @@ public static class GraphAttribute {
                 .setDefaultValue(50000)
                 .create()
                 .build();
+        public static final SchemaAttribute CONNECTION_MOTION = new SchemaAttribute.Builder(GraphElementType.META, FloatAttributeDescription.ATTRIBUTE_NAME, "motion")
```

Comment:
should this graph element type be `GRAPH` instead of `META`? If not, this should probably be moved to its own MetaAttributes class

Reply:
The reason that this is META is so that it would be hidden from the Attributes View; this will allow the users to modify the other attributes on the Attributes View even when the animation (which constantly updates this attribute) is running. Will move this out to its own MetaAttributes class and reference it as a meta attribute. This has the effect of not triggering GraphChangeEvent which updates the displayValue for the AnimatePauseAction, so I've added a direct call to displayValue() from updateValue().